sfcasts/crud-controller.md

```diff
@@ -1,171 +1,90 @@
+
+Then, below this, you can actually see all of the code that executes the pages. So `index` is our index page, `detail` is a detail show page, `edit` is the edit form.
+
+You can see the code that actually runs behind all of this, which is super useful when we're figuring out how to extend things. But... wait a second. If you scroll back up here to the configure methods, a few of these look familiar. Some of them *also* exist in the dashboard based controller class, specifically `configureAssets`, `configureCrud`, `configureActions`, and `configureFilters`. These all live side of `AbstractDashboardController`. There's `configureAssets`, and then further down here, CRUD actions and filters.
+
+So what's going on?
+
+Click "Questions" and look at the URL. It starts with `/admin` and then a bunch of query parameters. Turns out, everything EasyAdmin is handled via a single giant route. It all runs through the `DashboardController` route - the `/route` that's above the index. So when we go to the `QuestionCrudController`, it's actually matching this route here with extra query parameters to say which CRUD controller and which action runs. You can see `crudController` and `crudAction` right there.
+
+We're rendering `QuestionCrudController`, but in the context of our `DashboardController`. When we go to this page, in order to get the CRUD config, EasyAdmin first calls `configureCrud` on our dashboard. Then it calls `configureCrud` on our specific ride controllers. In this case, `QuestionCrudController` is super powerful. It means that we can override things on a dashboard level, like set some CRUD settings to apply to all sections or for one specific `crudController`.
+
+I'll prove it! Go back to `AbstractDashboardController`. In `configureCrud`, every area has four pages. I'll hold "cmd" and click to open this `Crud` thing. There's some constants on top here. Every CRUD section has an index page. That's this in 'edit' page, a 'new' page, and also a 'detail' page. Each page can then refer to a set of actions. These are links or buttons. For example, on the index page, right now, we have an action for editing and an action for deleting. We *could* add more actions or remove those actions.
+
+You can see how this is configured down in `configureActions`. This is for the dashboard, so this is the action configuration that applies to all of our sections. You can see that for the index page. It adds new "Edit" and "Delete" actions for the detail page. There's `EDIT`, `INDEX`, and `DELETE`. And if you're on the edit page, you have the actions `SAVE_AND_RETURN` and `SAVE_AND_CONTINUE`. One thing you'll notice if you look closely here is that we *do* have a detail page, but nobody actually links to it. You don't see an action calling `DETAIL` on any of these pages. So the page exists, but it's not really used out of the box. So let's add that. 
```

Comment:
I just noticed that perhaps the sentences "You can see that for the index page." and "It adds new..." should be a single sentence. I guess it depends on the inflection of the word "that"...